Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Calendar/Krypton.Toolkit.Suite.Extended.Calendar.csproj

```diff
@@ -51,5 +51,8 @@
     <DocumentationFile>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.Calendar\Krypton.Toolkit.Suite.Extended.Calendar.xml</DocumentationFile>
     <OutputPath>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.Calendar\</OutputPath>
   </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualBasic" />
```

Comment:
What's this for ?

Reply:
Some controls use VB APIs (possibly for .NET 3.5 compatibility)

Reply:
Eek.. That's a big file to pull in for a "Few" API's..
Can these not be put into the a compatibility class ?

Reply:
Actually, it may not be needed

Reply:
As I thought, resolved.

Reply:
Fixed by adding a compatibility method